ext/pdo_sqlite/sqlite_driver.c

```diff
@@ -226,6 +226,15 @@ static zend_string* sqlite_handle_quoter(pdo_dbh_t *dbh, const zend_string *unqu
 	if (ZSTR_LEN(unquoted) > (INT_MAX - 3) / 2) {
 		return NULL;
 	}
+	if(memchr(ZSTR_VAL(unquoted), '\0', ZSTR_LEN(unquoted)) != NULL) {
```

Comment:
Please use the standard `zend_str_has_nul_byte` API.
Also style nit: space after `if`.